### ISSUE TITLE
FR: Should be able to add barclamps from a node's page in web UI

### ISSUE THREAD

**Issue by aspiers** [crowbar/crowbar#909]
It's not a great UI experience to require the user to first navigate to the list of barclamps via the link at the top right before being able to create a proposal.  It would be nice if there was a button on the node view page for adding barclamps to that node.  This makes sense particularly for barclamps which are generally only applied to a single node; however in the multi-node barclamp case, if a proposal already existed for that barclamp, the UI should offer the choice of adding the node to the barclamp's existing proposal, or creating a new proposal.

**Comment by aspiers** [comment]
@bkutil @tboerger This is a very old issue but still valid I think.

**Comment by bkutil** [comment]
Agreed. This is a part of larger idea how setting up the deployment (role layout) of the cluster should look like, isn't it? I think it'd be good to create an epic for this.

**Comment by aspiers** [comment]
Yup, probably.  Have added to https://etherpad.nue.suse.com/p/crowbar-app-roadmap-2015

**Comment by bkutil** [comment]
Are you OK with closing this for now?

**Comment by aspiers** [comment]
What's the justification for closing it?

**Comment by bkutil** [comment]
We track it already on several places.

**Comment by aspiers** [comment]
@bkutil URLs?

**Comment by tboerger** [comment]
Sounds good
